src/browser/browser.ts

```diff
@@ -129,6 +129,16 @@ export class Browser {
     disableMemory?: boolean;
   }): Promise<Page> {
     const basePage = await this.browser.newPage();
+
+    await basePage.setRequestInterception(true);
+    basePage.on("request", (request) => {
+      if (request.url().startsWith("mailto:")) {
+        request.abort();
```

Comment:
my second question is, how does this interact with the state machine? will it just keep hammering the link?

Reply:
say more please